src/fides/connectors/models.py

```diff
@@ -45,12 +45,14 @@ class BigQueryConfig(BaseModel):
 
 class OktaConfig(BaseModel):
     """
-    The model for the connection config for Okta
+    The model for the connection config for Okta using OAuth2.
     """
 
-    # camel case matches okta client config model
+    # camelCase for frontend compatibility
```

Comment:
Not super critical here but in the future you can `alias` things in Pydantic - and, not that you'd want to, but you can even support different aliases for validation and serialization :smile: 
```
from pydantic import Field
class Foo(BaseModel):
   client_id: str = Field(default=..., alias='clientId')
````